modules/identity-provider-htpasswd-update-users.adoc

```diff
@@ -57,6 +57,22 @@ Deleting password for user <username>
 ----
 $ oc create secret generic htpass-secret --from-file=htpasswd=users.htpasswd --dry-run=client -o yaml -n openshift-config | oc replace -f -
 ----
++
+[TIP]
```

Comment:
there should not be an existing secret, is there? It would only replace the secret you created by the `oc create` command, but given this is an alternative, this should be fine, no?

Reply:
@stlaz So this section is for adding more users after you've already set up the HTPasswd IDP: https://deploy-preview-32987--osdocs.netlify.app/openshift-enterprise/latest/authentication/identity_providers/configuring-htpasswd-identity-provider.html#identity-provider-htpasswd-update-users_configuring-htpasswd-identity-provider

So you've already created the secret in a different section. And now you're replacing that secret (via the `oc replace -f -` part of the command) with a "new" secret with the updated users.passwd file. 

But maybe the YAML is indeed fine the way it is, and I just need to change the tip wording to say "to replace the secret" instead of "to create the secret".

Reply:
Oh I see, the yaml in this case is more or less the same as the output of `oc create secret generic htpass-secret --from-file=htpasswd=users.htpasswd --dry-run=client -o yaml -n openshift-config`. I think `oc apply` should still work.